### DISCUSSION TITLE
Would it be possible to begin designing the backend API and data model for this application?

### DISCUSSION THREAD

**Discussion by kcsaiganesh** [gauravsingh1281/Rentalog.in--Frontend#68] [Ideas]
@gauravsingh1281  I am proposing that we start designing the backend API and data model for this application. This is an important step in the development process, and it will help us to ensure that the application is well-designed 
 I would prefer to suggestions on as follows:

- Security on Authentications, Authurization for api requets, user based access controll(jwt/middlwares).
- api design/ firebase as SaaS.
- credentials managment

**Comment by gauravsingh1281** [comment]
@kcsaiganesh yes, go for it.

**Comment by kcsaiganesh** [comment]
@gauravsingh1281 I request you to create rentallog- parent folder to add a backend folder and current client in this repo , which makes app more easy and efficent to handle both backend and client.

**Reply by gauravsingh1281** [reply]
Ok
